molecule/agent-smoke-universal-ebpf/converge.yml.template

```diff
@@ -0,0 +1,18 @@
+---
+- name: Converge
+  hosts: all
+  strategy: free
+  become: true
+  roles:
+    - role: sysdig.agent.agent_install
+  vars:
+    configuration:
+      connection:
+        access_key: ${AGENT_ACCESS_KEY}
+        custom_collector:
+          url: ${COLLECTOR_URL}
+          port: ${COLLECTOR_PORT}
+      agent:
+        driver:
+          install_build_dependencies: false
```

Comment:
Is it needed?

Reply:
(I suppose is suggested but not mandatory)

Reply:
@mavimo, build dependencies are completely unnecessary for Universal eBPF.  The eBPF object is built alongside the rest of the agent and embedded in the `dragent` binary.

Reply:
@mprzybylski
> build dependencies are completely unnecessary for Universal eBPF 

I know my question if is something that we need to specify or if we are doing that just for the sake of velocity in the test.

BTW since is unnecessary we may consider to notify the customer that we suggest to move to "false" when `universal_ebpf` is selected, since at the moment there are no places where we are reporting this information 😄

Reply:
> ...my question if is something that we need to specify or if we are doing that just for the sake of velocity in the test.

Just for the sake of velocity.  _Having_ the kmod / legacy eBPF build dependencies doesn't break anything.